x-pack/platform/packages/shared/kbn-failure-store-modal/src/components/failure_store_modal/failure_store_modal.tsx

```diff
@@ -44,18 +44,29 @@ export interface FailureStoreFormProps {
 interface FailureStoreFormData {
   failureStoreEnabled: boolean;
   customRetentionPeriod?: string;
+  inherit?: object;
```

Comment:
2 Questions 

1. is this actually used/supposed to be used anywhere? Because I don't see it passed I only see the other 2 params used

2.
is this type intentionally this wide? `object` also includes a function

<img width="276" height="73" alt="Image" src="https://github.com/user-attachments/assets/3cf87270-485c-4759-8802-f15ccf4063f3" />

If we want a plain object that can contain anything and we don't know what that we should just explicitly use 


```suggestion
  inherit?: Record<PropertyKey, unknown>;
```

OR better use a generic to make each callback unique to the call

Reply:
> The modal doesn't need to know how the consumer plugin will use this information

Generally true, but since you are the one implementing both it arguably makes it a little easier to reason about the change when definition and consumption are colocated in the same PR.

But  up to you. Will resolve now. Thanks for clarifying